systemtests/tests/python-bareos/test_list_command.py

```diff
@@ -58,10 +58,12 @@ def test_list_jobs(self):
             **self.director_extra_options,
         )
 
-        director.call("run job=backup-bareos-fd yes")
-        director.call("wait")
-        director.call("restore client=bareos-fd fileset=SelfTest select all done yes")
-        director.call("wait")
+        backup_jobname = "backup-bareos-fd"
+        backup_jobid = self.get_backup_jobid(director, backup_jobname, level="Full")
+        self.run_restore(director, client="bareos-fd", jobid=backup_jobid, wait=True)
+        backup_jobid_incremental = self.get_backup_jobid(
+            director, backup_jobname, level="Incremental"
+        )
```

Comment:
previously we didnt run an incremental here.  The incremental jobid is also not used.  What changed that we now want to run this ?

Reply:
We need the incremental for some later tests in this function. Running it has just been moved to an earlier point in time.